packages/endpoints-core/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@ossinsight/endpoints-core",
```

Comment:
Maybe we can call it '@ossinsight/data-service'? 😄

Reply:
changed